components/server/ee/src/workspace/gitpod-server-impl.ts

```diff
@@ -2275,6 +2276,12 @@ export class GitpodServerEEImpl extends GitpodServerImpl {
         await this.guardCostCenterAccess(ctx, user.id, attrId, "get");
 
         const { costCenter } = await this.usageService.getCostCenter({ attributionId });
+        return this.translateCostCenter(costCenter);
+    }
+
+    private translateCostCenter(
+        costCenter: CostCenter | undefined,
+    ): CostCenterJSON | PromiseLike<CostCenterJSON | undefined> | undefined {
```

Comment:
nit: wouldn't a simple `Promise<CostCenterJSON | undefined>` as result be simpler to read? 
also `CostCenterJSON` seems to be imported from `src`.
also 😢  that we need to redefine the shape.

Reply:
sidebar: search for `from "@gitpod/gitpod-protocol/src/` reveals a couple of findings. maybe something is broken with vscode/TS imports (again) 🤔  or one need to rebuild the protocol project first, to get the suggestion for `/lib/` as well.